sample/07-geopip-blocking/src/main.ts

```diff
@@ -0,0 +1,11 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app.module';
+import { GeoIPInterceptor } from './geoip.interceptor';
```

Comment:
import from package.
```suggestion
import { GeoIPInterceptor } from '@samagra-x/stencil';
```